openedx_webhooks/github/handlers/bouncer/__init__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import logging
+
+from ....services.rq import q as _q
+from ...models import GithubHeader
+from .rules import RULES
+
+logger = logging.getLogger(__name__)
+
+
+def bounce(raw_headers, event, q=_q, rules=RULES):
+    """
+    Determines whether the event needs to be processed.
+
+    Args:
+        raw_headers (flask.Request.headers)
+        event (dict): The parsed event payload
+        q (rq.Queue)
+    """
+    event_type = GithubHeader(raw_headers).event_type
+
+    for rule in rules:
+        rejected, message = rule.check(event_type, event)
+        if rejected:
+            return message
+
+    q.enqueue('openedx_webhooks.github.handlers.process', event_type, event)
```

Comment:
You get all three of those benefits from functions (or classes with methods).  Queueing is for other things (doing work on other machines, deferring work, responding quickly in spite of long work, retrying).  We don't need any of those things on the second and third queuing here.